BookPlayer/Import/ImportManager.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ImportManager.swift
+//  BookPlayer
+//
+//  Created by Gianni Carlo on 9/10/18.
+//  Copyright © 2018 Tortuga Power. All rights reserved.
+//
+
+import BookPlayerKit
+import Combine
+import Foundation
+
+/**
+ Handles the creation of ImportOperation objects.
+ It waits a specified time wherein new files may be added before the operation is created
+ */
+class ImportManager {
```

Comment:
```suggestion
final class ImportManager {
```
To avoid DynamicDispatch